src/directives/partials/popup.html

```diff
@@ -0,0 +1,5 @@
+<h4 class="popover-title ngeo-popup-title">
+  <span translate>{{options.title}}</span>
```

Comment:
Also, I'm wondering whether we should start using `ng-bind` instead of `{{…}}` to avoid displaying ugly expressions in the UI at load time.